### PR TITLE
create rds engine vars

### PR DESCRIPTION
**Pull request by domjngo**

### CODE REVIEW

File: rds/rds.tf [hunk 1]
```diff
@@ -20,28 +20,40 @@ resource "aws_db_instance" "wp_db_main" {
...
 
     tags = {
-        Name            = "${var.service}-wp-${var.environment}-ma"
-        Service         = var.service
-        Environment     = var.environment
-        CostCentre      = var.cost_centre
-        Owner           = var.owner
-        CreatedBy       = var.created_by
-        Terraform       = true
+        Name        = "${var.service}-wp-${var.environment}-ma"
```
Comment (milh0use) [inline rds/rds.tf]: Did you mean to leave this name unmodified? It's different from the actual `identifier` above?

File: rds/rds.tf [hunk 2]
```diff
@@ -52,13 +64,13 @@ resource "aws_db_instance" "wp_db_replica" {
...
 
     tags = {
-        Name            = "${var.service}-wp-${var.environment}-rr"
-        Service         = var.service
-        Environment     = var.environment
-        CostCentre      = var.cost_centre
-        Owner           = var.owner
-        CreatedBy       = var.created_by
-        Terraform       = true
+        Name        = "${var.service}-wp-${var.environment}-rr"
```
Comment (milh0use) [inline rds/rds.tf]: Likewise, `identifier` is now different?

**Review by milh0use — COMMENTED** [review]
Sorry, I accidentally clicked the "Start Review" button instead of the "Add single comment" button. Not sure what the difference is but I added two comments on possible mistakes.

File: rds/rds.tf (hunk 1 above)
Comment (domjngo) [reply rds/rds.tf]: I don't think it matters, but I'll change it.

**Review by milh0use — APPROVED** [review]

**Pull request merged**